package-lock.json

```diff
@@ -46118,6 +46002,287 @@
+        "interruptor": "^1.0.1",
+        "system-ca": "^2.0.1",
+        "web-worker": "^1.3.0"
+      },
+      "engines": {
+        "node": ">=14.15.1"
+      }
+    },
+    "packages/compass-shell/node_modules/@mongosh/shell-api": {
+      "version": "3.12.0",
+      "resolved": "https://registry.npmjs.org/@mongosh/shell-api/-/shell-api-3.12.0.tgz",
+      "integrity": "sha512-3LYoiq12LbexH9f+11PBQz0J2nYAdnXrhgnW3S994hQgEKghPJb/qvD/9KiaMBRWVhTuawmdplGaFRLIr3oRfw==",
+      "license": "Apache-2.0",
+      "dependencies": {
+        "@babel/core": "^7.26.10",
+        "@babel/types": "^7.26.10",
+        "@mongosh/arg-parser": "^3.10.3",
+        "@mongosh/errors": "2.4.0",
+        "@mongosh/history": "2.4.6",
+        "@mongosh/i18n": "^2.13.1",
+        "@mongosh/service-provider-core": "3.3.3",
+        "mongodb-redact": "^1.1.5",
+        "mongodb-schema": "^12.6.2"
+      },
+      "engines": {
+        "node": ">=14.15.1"
+      }
+    },
+    "packages/compass-shell/node_modules/@mongosh/shell-evaluator": {
+      "version": "3.12.0",
+      "resolved": "https://registry.npmjs.org/@mongosh/shell-evaluator/-/shell-evaluator-3.12.0.tgz",
+      "integrity": "sha512-8p0fVRQ2TodANKUuYLCY/mtB3LEzRyo/ZtskKTOwzVHwTNgdkBuwoBcKoSOLcSLihO7ZAWumPWlFNh+ukECmAw==",
+      "license": "Apache-2.0",
+      "dependencies": {
+        "@mongosh/async-rewriter2": "2.4.8",
+        "@mongosh/history": "2.4.6",
+        "@mongosh/shell-api": "^3.12.0"
+      },
+      "engines": {
+        "node": ">=14.15.1"
+      }
+    },
+    "packages/compass-shell/node_modules/@mongosh/types": {
+      "version": "3.7.0",
+      "resolved": "https://registry.npmjs.org/@mongosh/types/-/types-3.7.0.tgz",
+      "integrity": "sha512-2eo5y5GlYz/vbz0E/00rBY+xj9Gw4VIq7fe/KAkva3Os1nsFUkEWUMKBA2oSfdbYEQmxia/huVwhBElqmHqmZw==",
+      "license": "Apache-2.0",
+      "dependencies": {
+        "@mongodb-js/devtools-connect": "^3.4.1"
+      },
+      "engines": {
+        "node": ">=14.15.1"
+      }
+    },
+    "packages/compass-shell/node_modules/ansi-regex": {
+      "version": "5.0.1",
+      "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-5.0.1.tgz",
+      "integrity": "sha512-quJQXlTSUGL2LH9SUXo8VwsY4soanhgo6LNSm84E1LBcE8s3O0wpdiRzyR9z/ZZJMlMWv37qOOb9pdJlMUEKFQ==",
+      "license": "MIT",
+      "optional": true,
+      "engines": {
+        "node": ">=8"
+      }
+    },
+    "packages/compass-shell/node_modules/cliui": {
+      "version": "8.0.1",
+      "resolved": "https://registry.npmjs.org/cliui/-/cliui-8.0.1.tgz",
+      "integrity": "sha512-BSeNnyus75C4//NQ9gQt1/csTXyo/8Sb+afLAkzAptFuMsod9HFokGNudZpi/oQV73hnVK+sR+5PVRMd+Dr7YQ==",
+      "license": "ISC",
+      "optional": true,
+      "dependencies": {
+        "string-width": "^4.2.0",
+        "strip-ansi": "^6.0.1",
+        "wrap-ansi": "^7.0.0"
+      },
+      "engines": {
+        "node": ">=12"
+      }
+    },
+    "packages/compass-shell/node_modules/mongodb-schema": {
```

Comment:
Kinda weird it pulled a different version of mongodb-schema for shell only especially as this is a minor version diff with what we have in the repo and caret should cover it. Do you mind including updating schema to latest in your update here too?

Reply:
Nice catch, updated.